serverless-now/package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "start": "sls offline",
     "debug": "export SLS_DEBUG=* && node --inspect ./node_modules/.bin/sls offline",
-    "test": "sls invoke test",
+    "test": "sls invoke test --aws-profile saml",
     "testw": "nodemon --watch . --exec 'sls invoke test || true'",
-    "deploy": "npm run test && sls deploy"
+    "deploy-quick": "sls deploy --stage sandbox --aws-profile saml",
```

Comment:
Any reason why you have this?

Reply:
Because the testing was timing out so you suggested deploying and then re-running the tests to test accessing env variables. I made it command so I don't have to type it out every time and thought it might be useful to have a deploy option without running tests.

Reply:
Hmm maybe I'm not understanding correctly... but my question is, why does the `deploy` run tests still? 🤔 

What do you think? I guess we don't want to just be reliant on Jenkins as being our only way to deploy in case it crashes so it's good to have it, but then shouldn't the Jenkins job be calling `npm run deploy` instead of doing this:

https://github.com/UoA-eResearch/hub-stack/blob/3228778363b658de06fe37c4b6d84274719e66e9/Jenkinsfile#L204

That way we only have to maintain the deploy command in one place, same with this line here:

https://github.com/UoA-eResearch/hub-stack/blob/3228778363b658de06fe37c4b6d84274719e66e9/Jenkinsfile#L141

If you have a look at my stages I'm always just calling the existing NPM scripts. Thoughts?